packages/forma-36-react-components/src/components/Workbench/Workbench.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+import cn from 'classnames';
+import Heading from '../Typography/Heading';
+
+const styles = require('./Workbench.css');
+
+interface WorkbenchHeaderProps {
+  title?: string;
+  icon?: React.ReactElement;
+  actions?: React.ReactElement;
+}
+
+export function WorkbenchHeader(props: WorkbenchHeaderProps) {
+  return (
+    <div className={styles['Workbench__header']}>
+      {props.icon ? (
+        <div className={styles['Workbench__header-icon']}>{props.icon}</div>
+      ) : null}
+      {props.title ? (
+        <Heading
+          className={styles['Workbench__header-title']}
+          testId="workbench-title"
+        >
+          {props.title}
+        </Heading>
+      ) : null}
+      {props.actions ? (
+        <div className={styles['Workbench__header-actions']}>
+          {props.actions}
+        </div>
+      ) : null}
+    </div>
+  );
+}
+
+interface WorkbenchSidebarProps {
+  children: React.ReactNode;
+  position: 'left' | 'right';
+}
+
+export function WorkbenchSidebar(props: WorkbenchSidebarProps) {
+  return (
+    <div
+      className={cn(styles['Workbench__sidebar'], {
+        [styles['Workbench__sidebar--right']]: props.position === 'right',
+        [styles['Workbench__sidebar--left']]: props.position === 'left',
+      })}
+    >
+      {props.children}
+    </div>
+  );
+}
+
+interface WorkbenchContentProps {
+  children: React.ReactNode;
+  type: 'default' | 'text' | 'full';
+  testId: string;
+  leftSidebar?: React.ReactNode;
+  rightSidebar?: React.ReactNode;
```

Comment:
Did you consider making those not render props but rather just the `<WorkbenchSidebar>` component that can be used together with `<Workbench.Content>` You are then free to use your own styles. It should respect the grid/flexbox  regardless.

```
<Workbench>
  <WorkbenchHeader />
  <WorkbenchSidebar position="left" />
  <WorkbenchContent />
  <WorkbenchSidebar><MyCustomHugeSlideInSidebar /></WorkbenchSidebar>
</Workbench>
```
It would also be possible to have multiple sidebars on one side i.e. Jira's multi-level side nav. 

This would require the `Workbench--wrapper` to be outside of `<WorkbenchContent>`

Reply:
@guilebarbosa In this case implementation would be something like.

1. Find `WorbenchHeader` and `rest` components in `children`
2. Wrap `rest` (all sidebars and contents) with  a wrapper `div`.

right?

Reply:
I'm 👍 for making this changes if @wichniowski and @m10l  are agree.

Reply:
Sounds good to me!